x-pack/plugins/task_manager/server/task_store.ts

```diff
@@ -302,23 +307,13 @@ export class TaskStore {
    */
   public async bulkUpdate(
     docs: ConcreteTaskInstance[],
-    { validate, excludeLargeFields = false }: BulkUpdateOpts
```

Comment:
these changes were originally made so we could exclude the `state` and `params` from being updated in the `mget` claim strategy but they are not needed anymore since we can do real partial updates through the ES client.